automation_tools/scripts/ga-migration/main.py

```diff
@@ -135,6 +136,12 @@ def migrate_repo(path):
     # Remove bak files
     delete_file(path + "*.bak")
 
+    # Add newline at EOF
+    add_newline_at_end(path + ".github/workflows/tests.yml")
+    add_newline_at_end(path + "MANIFEST.in")
+    add_newline_at_end(path + "pytest.ini")
```

Comment:
Not sure if this one is needed, I think `run-tests.sh` is the only one that has EOF issues.